Makefile

```diff
@@ -17,6 +17,12 @@ clean:
 
 composer:
 	docker-compose run --rm composer bash -c "composer install --no-scripts"
+	touch vendor/simplesamlphp/simplesamlphp/config/authsources.php
```

Comment:
Do you still need these `touch` commands since you reverted the `vendor` volume change?

Reply:
Yeah, those `touch` commands are what enabled me to revert the `vendor` volume change. I think it has to do with the way we're volume-mounting individual files into the vendor/simplesamlphp folder AND we're volume-mounting the entire vendor folder... those files end up being created as directories if they don't already exist, so I need to `touch` them after installing the composer dependencies.